Podfile.lock

```diff
@@ -66,7 +66,7 @@ PODS:
     - WordPressKit (~> 4.18-beta)
     - WordPressShared (~> 1.12-beta)
     - WordPressUI (~> 1.7-beta)
-  - WordPressKit (4.24.0):
+  - WordPressKit (4.25.0-beta.3):
```

Comment:
Hey @ctarda, Just wanted to triple-check with you that this was intentional. This bump means that this PR that targets the code-frozen 5.8 branch will now include code of `WordPressKit` from _after_ the code freeze of 5.8, potentially some that was added to provide features and changes aimed for / used only… in the current `develop` branches WordPress or Woo.

Strictly speaking, if a fix on the current release branch needs one of our dependency to be updated to add a specific fix on it, it's usually more appropriate to cut a hotfix branch from 4.24.0 (the version of the dependency we used when `release/5.8` was code-frozen) just to add the specific fix you need on top, and then make a 4.24.1 and point to it.

As you updated WordPressKit to `4.25.0-beta.3` here, that might mean that your fix on release/5.8 will not only include whatever you changed and added to `WordPressKit` specifically to fix this issue, but also any other change that was added between `4.24.0` and `4.25.0-beta.2`.

That being said, while it would be a bit dangerous to include all that if that was a hotfix on an already-published production build, that might not be that big of a deal in the case of a fix on a `release/5.8` like here that was not out yet and will still have time to be beta-tested before your fix is released to prod, but that's still something to be wary of.
I think it's ok to keep that **if you're entirely sure that whatever unrelated changes between `4.24.0` and `4.25.0-beta.2` you brought (in addition to whatever was in `4.25.0-beta.3` you needed here) didn't include neither a regression, nor a new feature that was meant for `develop` or for the future `5.9` only**.

For reference, changes brought by this dependency bump seems include: https://github.com/wordpress-mobile/WordPressKit-iOS/issues/325 (4.25.0-beta.1), https://github.com/wordpress-mobile/WordPressKit-iOS/issues/327 (4.25.0-beta.2), and https://github.com/wordpress-mobile/WordPressKit-iOS/issues/326 (4.25.0-beta.3)

Reply:
Good catch @AliSoftware 

It was not on purpose. I think what might have happened is:
* At one moment in time, while working on the PR, I [bumped WPAuthenticator](https://github.com/woocommerce/woocommerce-ios/pull/3483/commits/7741e305b1313096b7e9775350f6d3b78d8c1c68)
* I reverted that bump back later, but I guess that didn't revert the bump to WPKit

Anyway, I agree with this:
> while it would be a bit dangerous to include all that if that was a hotfix on an already-published production build, that might not be that big of a deal in the case of a fix on a release/5.8 l

And the three PRs in WPKit you linked to do not seem to affect WooCommerce. 

So I think it's okay to keep it

Reply:
Thanks for the answer. Actually keeping that change here will require me to stabilize a 4.25.0 version at the time of actual release on Friday – because we never ship dependencies in beta state to the AppStore – which doesn't seem right (will include even more new `-beta.N` changes, untested with WC 5.8). I'd prefer that we revert to 4.24 if your PR didn't mean any new fix or feature from WPKit in the first place.

Reply:
Sure. Reverted in #3489